### DISCUSSION TITLE
How to use Acoustic Echo Cancellation (AEC) without webrtc_aec module

### DISCUSSION THREAD

**Discussion by GOBIKRISHNAN-R** [baresip/baresip#1829] [Q&A]
From the baresip wiki, I tried the echo cancellation with pulse module. 

https://github.com/baresip/baresip/wiki/Acoustic-Echo-Cancellation-(AEC)

If I'm using the webrtc_aec module, the echo cancellation is working. But my concern is, I just followed the wiki page and i simply enabled the pulse module alone. Now the echo cancellation is not working.

Working case = pulse + webrtc_aec
Not working case  = pulse only

Can i get any suggestions?

**Comment by sreimers** [comment]
Is pulse configured like this with echo-cancel module? https://github.com/baresip/baresip/wiki/Using-PulseAudio:-module-echo-cancel

**Reply by GOBIKRISHNAN-R** [reply]
Yes sreimers, I followed the above mentioned link only

**Comment by jobo-zt** [comment]
webrtc AEC module and its required dependencies module for windows
git clone -b baresip-libwebrtc https://github.com/jobo-zt/libwebrtc.git
cmd.exe -> mkdir build -> cd build -> cmake ..
